volplugin/handlers.go

```diff
@@ -313,13 +210,14 @@ func (dc *DaemonConfig) mount(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := applyCGroupRateLimit(volConfig, mc); err != nil {
+	if err := applyCGroupRateLimit(volConfig.RuntimeOptions, mc); err != nil {
 		httpError(w, "Applying cgroups", err)
 		return
 	}
 
 	go dc.Client.HeartbeatMount(dc.Global.TTL, ut, dc.Client.AddStopChan(uc.Request.Name))
 
+	go dc.startRuntimePoll(volConfig.String(), mc)
```

Comment:
In addition to handling mount, in case of volplugin restart we may need to poll for already mounted volumes as well, right?